### PR TITLE
Add amorvan/responders-bundle

### PR DESCRIPTION
**Pull request by AurelienMo**
| Q             | A
| ------------- | ---
| License       | MIT

For reference:
[https://github.com/AurelienMo/RespondersBundle](https://github.com/AurelienMo/RespondersBundle)

### CODE REVIEW

**Comment by OskarStark** [comment]
CleanShot 2021-03-17 at 16 35 39

It looks like your email address is not associated with your Github Account

**Comment by AurelienMo** [comment]
@OskarStark Failed on build travis, I don't understand why. If I create a project on local environment (same process as Travis), no errors occured :(

**Comment by AurelienMo** [comment]
> CleanShot 2021-03-17 at 16 35 39
> 
> It looks like your email address is not associated with your Github Account

It's ok

**Comment by AurelienMo** [comment]
@OskarStark CI travis ok :+1:

File: amorvan/responders-bundle/4.0/config/packages/amorvan_responders_bundle.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+services:
+    _defaults:
+        autowire: true
+
+    Morvan\Bundle\RespondersBundle\Responders: null
```
Comment (Nyholm) [inline amorvan/responders-bundle/4.0/config/packages/amorvan_responders_bundle.yaml:5]: You bundle should register these services in https://github.com/AurelienMo/RespondersBundle/blob/development/Resources/config/services.xml

You dont need a recipe =) 
Things will work anyways.

**Review by Nyholm — CHANGES_REQUESTED** [review]

**Pull request closed without merging**